tools/osbuild-mpp

```diff
@@ -771,14 +772,16 @@ class Partition:
                  size: int = None,
                  bootable: bool = False,
                  name: str = None,
-                 uuid: str = None):
+                 uuid: str = None,
+                 attrs: [int] = None):
```

Comment:
Is `[int]` a newly supported format for type annotations or should this be `List[int]`?